src/UnityExtension/Assets/Editor/GitHub.Unity/UI/PUblishView.cs

```diff
@@ -0,0 +1,242 @@
+                            {
+                                Logger.Warning("Unable to get list of organizations");
+                                return;
+                            }
+
+                            Logger.Trace("Loaded {0} organizations", organizations.Count);
+
+                            var organizationLogins = organizations
+                                .OrderBy(organization => organization.Login)
+                                .Select(organization => organization.Login);
+
+                            owners = owners.Union(organizationLogins).ToArray();
+                        });
+                    });
+                }
+                else
+                {
+                    Logger.Warning("No Keychain connections to use");
+                }
+            }
+            catch (Exception e)
+            {
+                Logger.Error(e, "Error PopulateView & GetOrganizations");
+                throw;
+            }
+        }
+
+        public override void OnGUI()
+        {
+            GUILayout.BeginHorizontal(Styles.AuthHeaderBoxStyle);
+            {
+                GUILayout.BeginVertical(GUILayout.Width(16));
+                {
+                    GUILayout.Space(9);
+                    GUILayout.Label(Styles.BigLogo, GUILayout.Height(20), GUILayout.Width(20));
+                }
+                GUILayout.EndVertical();
+
+                GUILayout.BeginVertical();
+                {
+                    GUILayout.Space(11);
+                    GUILayout.Label(Title, EditorStyles.boldLabel);
+                }
+                GUILayout.EndVertical();
+            }
+            GUILayout.EndHorizontal();
+
+            GUILayout.Space(5);
+
+            GUILayout.BeginHorizontal();
+            {
+                GUILayout.BeginVertical();
+                {
+                    GUILayout.Label("Owner");
+
+                    GUI.enabled = !isBusy;
+                    selectedOwner = EditorGUILayout.Popup(0, owners);
+                    GUI.enabled = true;
+                }
+                GUILayout.EndVertical();
+
+                GUILayout.BeginVertical(GUILayout.Width(8));
+                {
+                    GUILayout.Space(20);
+                    GUILayout.Label("/");
+                }
+                GUILayout.EndVertical();
+
+                GUILayout.BeginVertical();
+                {
+                    GUILayout.Label("Repository Name");
+                    GUI.enabled = !isBusy;
+                    repoName = EditorGUILayout.TextField(repoName);
+                    GUI.enabled = true;
+                }
+                GUILayout.EndVertical();
+            }
+            GUILayout.EndHorizontal();
+
+            GUILayout.Label("Description");
+            GUI.enabled = !isBusy;
+            repoDescription = EditorGUILayout.TextField(repoDescription);
+            GUI.enabled = true;
+            GUILayout.Space(5);
```

Comment:
Padding/margin spacing should be set in `const` fields in the Styles so we don't have to track them down everywhere in code. There's a bunch throughout the UI here.